content/docs/0.7.x/operate/install/index.md

```diff
@@ -222,6 +222,32 @@ Use this page if you have already Keptn experience and would like to install it
 
 ## Install Keptn CLI
 
+### MacOS
+
+Keptn CLI is also available as Homebrew package for macOS.
+
+#### Install
+
+This will automatically fetch the latest Binary from Homebrew
+
+```sh
+brew install keptn
+```
+
+#### Updating
+
+```sh
+brew upgrade keptn
+```
+
+#### Uninstall
+
+```sh
+brew uninstall keptn
+```
+
+### Others
+
 Every Keptn release provides binaries for the Keptn CLI. These binaries are available for Linux, macOS, and Windows.
 
```

Comment:
Thanks for your contribution.

To be honest, I think this leaves the impression that Keptn CLI is primary available for MacOS.

I would rather like to have it the other way round, and provide the MacOS way via brew below the GitHub Releases download page for now.